editor.planx.uk/src/@planx/components/DateInput/model.ts

```diff
@@ -71,29 +72,41 @@ export const parseDate = (date?: string) => {
 export const dateSchema = () => {
   return string()
     .test("missing day", "Date must include a day", (date?: string) => {
+      if (!date) return true;
```

Comment:
I spent a while looking into this - there's not a hierarchical way to run this check once and first - it needs to be repeated. 

Pretty tedious / error prone! If this is not included, we'll run validations on optional fields (which will always fail).